pyvcloud/vcd/vapp.py

```diff
@@ -124,6 +124,18 @@ def set_lease(self, deployment_lease=0, storage_lease=0):
             new_section,
             EntityType.LEASE_SETTINGS.value)
 
+    def change_owner(self, href):
```

Comment:
Please add a doc string to every new functionality method you introduce in pyvcloud.